app/controllers/agent/eligibility/CovidCannotSignUpController.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.agent.eligibility
+
+import auth.agent.AuthenticatedController
+import config.AppConfig
+import javax.inject.{Inject, Singleton}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import services.AuthService
+import views.html.agent.covid_cannot_sign_up
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class CovidCannotSignUpController @Inject()(val authService: AuthService)
+                                           (implicit val ec: ExecutionContext, mcc: MessagesControllerComponents,
+                                            appConfig: AppConfig)
+  extends AuthenticatedController {
+
+  val show: Action[AnyContent] = Authenticated.async { implicit request =>
+    implicit user =>
+      Future.successful(
+        Ok(covid_cannot_sign_up(postAction = controllers.agent.eligibility.routes.Covid19ClaimCheckController.show(), backUrl))
+      )
+  }
```

Comment:
```suggestion
  val show: Action[AnyContent] = Authenticated { implicit request =>
    implicit user =>
      
        Ok(covid_cannot_sign_up(postAction = controllers.agent.eligibility.routes.Covid19ClaimCheckController.show(), backUrl))
      
  }
```
same as previous comment